recipes/ca-certificates/bld.bat

```diff
@@ -0,0 +1,16 @@
+:: Windows really uses a different store of certificates (see
+::   http://superuser.com/questions/411909/where-is-the-certificate-folder-in-windows-7)
+
+:: The certs here are probably only useful for unix-type apps, such as git, that don't necessarily
+::    respect the windows way of doing things.
+
+:: version at time of writing is 1.25
+curl -O https://raw.githubusercontent.com/curl/curl/master/lib/mk-ca-bundle.pl
+if ( $($(CertUtil -hashfile mk-ca-bundle.pl SHA256)[1] -replace " ","") -eq "6fbdd1c76a41b7ab41cd616b19e52522e2b7efb636120950f053ef2c22de44af" ) { echo "mk-ca-bundle download ok" } else {echo "mk-ca-bundle.pl checksum bad.  Has it changed?" && exit 1}
```

Comment:
> Have you looked at the downloaded file? Does it look sane? Are newlines changing and changing the sha?

So, the main trick here (at least for me) is I don't really have a Windows VM to work with. I can download the files on my Mac, but that doesn't really tell me anything about what is going on with Windows. If you want to try or can help me setup a VM for Windows, I'll happily try playing with it there.

However, I wouldn't really expect the file content to vary over platforms. My expectation is curl simply downloads the bytes regardless of file type or platform. Doing something different on different platforms would be subject to criticism.

> Curl here might need a -L flag, if github has any redirection going on. However, I would expect that to be consistent across platforms.

Sure. Willing to try anything you have in mind.

Though I would agree. Redirection doesn't seem to happen when visiting it in the browser and doesn't affect the Mac and Linux cases. Looking at the file on my Mac they seem to have what I would expect them to have.